src/java/org/apache/cassandra/db/streaming/ComponentManifest.java

```diff
@@ -106,6 +106,14 @@ public int hashCode()
         return components.hashCode();
     }
 
```

Comment:
nit: Outside this patch, but the JavaDoc for `ComponentManifest` is in the wrong place, and the imports are all over the place.